test/l10n/localizeTimestamp.js

```diff
@@ -57,12 +57,20 @@ describe('l10n/localizeTimestamp:', () => {
       assert.equal(format(Date.now() - 1, 'es-mx, ru'), 'hace unos segundos');
     });
 
+    it('returns a fallback language if called with unsupported language variations', () => {
+      assert.equal(format(Date.now() - 1, 'de, fr;q=0.8, ru;q=0.5'), 'несколько секунд назад');
```

Comment:
In this test are both Danish and French unsupported?

Reply:
Yep. (well except it's German not Danish, but yep)